### ISSUE TITLE
Update develop branch

### ISSUE THREAD

**Issue by raxod502** [raxod502/straight.el#704]
I think it's about time to do a general merge of master into develop, since we now have commits from September 2020 that are not available on master. @progfolio, you have the most context on recent development; what should be the cutoff point for which commits are merged into master? I usually default to a heuristic of anything older than a month (e.g. https://github.com/raxod502/straight.el/commit/ac4df5c in this case), but I also try to give more time for changes that I think are risky.

**Comment by progfolio** [comment]
I'd say we're good up to the current head of "develop". The commits after ac4df5c are relatively low-risk:

```
962d236 develop origin/develop Merge pull request #702 from raxod502/outline-regexp
4ddc3e2 Also move "straight.el ends here" footer
e36e9a9 [raxod502/selectrum#437] Use default outline-regexp
c94516d Include ox-odt styles when building org (#699)
0f9b828 straight--select-package: Add generic candidate filter. (#696)
dab2c6f straight--select-packge: fix for-build filter, candidate order (#695)
f2cb888 straight--directory-files: remove ".", "..", ".git" dirs when FULL paths (#693)
dedae21 straight-vc-git--ensure-remote: use git config to determine URL (#690)
```
I assume a good deal of our users are on Doom which defaults to the "develop" branch, too.

**Comment by raxod502** [comment]
Merged it. Not sure why there were so many conflicts, but I felt it was safe to resolve all of them in favor of develop.
